runtime/v2/bundle.go

```diff
@@ -101,7 +100,7 @@ func NewBundle(ctx context.Context, root, state, id string, spec []byte) (b *Bun
 		return nil, err
 	}
 	// write the spec to the bundle
-	err = os.WriteFile(filepath.Join(b.Path, configFilename), spec, 0666)
+	err = os.WriteFile(filepath.Join(b.Path, oci.ConfigFilename), spec, 0666)
```

Comment:
This is a reduced set of the changes from https://github.com/containerd/containerd/blob/9e34b8b441d7cf64a21d2073c3fbe7c5c15bc585/runtime/v2/bundle.go#L107-L115 (code diverged on main), only taking the `oci.ConfigFilename` from that commit.